templates/_header.phtml

```diff
@@ -17,7 +17,7 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
-  <title>Godot Asset Library</title>
+  <title><?php if(!empty($data['title'])){ echo(esc($data['title'].' - ')); } ?>Godot Asset Library</title>
```

Comment:
Just a note (but it would be fixed anyway later), we don't use `echo` as a function throughout the code :smile: